apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -74,13 +79,30 @@ const CommitsAndReviewTab = props => {
     }
   };
 
-  const closeReview = async () => {
+  const handleCloseReview = async (onSuccess, onFailure) => {
     try {
-      const closedReview = await dataApi.closeReview(openReviewData);
+      const closedReview = await dataApi.closeReview(openReviewData.id);
       setTimelineData([...timelineData, closedReview]);
       setOpenReviewData(null);
+      setIsReadOnlyWorkspace(false);
+      onSuccess();
     } catch (err) {
-      // TODO: what happens when review fails to close
+      console.log(err);
```

Comment:
should this log be here?

Reply:
We log caught errors to the console in many components. I'm not sure if we have a stance on that as an engineering team, I decided to leave the console here for debugging purposes